doc/reference/config.rst

```diff
@@ -31,3 +31,10 @@ Configuration Options (:mod:`pydicom.config`)
    APPLY_J2K_CORRECTIONS
    INVALID_KEY_BEHAVIOR
    INVALID_KEYWORD_BEHAVIOR
+   NO_VALIDATION
+   WARN_ON_ERROR
+   RAISE_ON_ERROR
+
+.. autoclass:: Settings
+   :exclude-members: __init__, reading_validation_mode
```

Comment:
I put this here separately to exclude these members, but this makes it visible only in the page for the `config` module, not in the overview page. Maybe there is a better way...